LabeledIconButton/LabeledIconButton.module.less

```diff
@@ -54,17 +271,102 @@
 			.label {
 				color: @agate-labelediconbutton-label-focus-color;
 			}
+
+			.button {
+				&:before {
+					content: @agate-labelediconbutton-before-content;
+					background: @agate-gradient;
```

Comment:
Maybe a new variable should be created here. I haven't seen so far `@agate-gradient` used directly in a component's css